SECURITY.md

```diff
@@ -30,9 +52,11 @@ Extensions are sandboxed from the host system by operating via APIs.  This secur
 
 ## Communications security (COMSEC)
 
-Plugins are loaded into the same java virtual machine instance allowing communicate to OpenSearch through in process java APIs.  Plugins can issue API requests to the OpenSearch hosts reusing the standard node-to-node communications, internally called the transport client.
+Plugins are loaded into the same java virtual machine instance allowing communication to OpenSearch through in-process java APIs, although this behavior will be deprecated over time in favor of using a common API endpoint.  Plugins can issue API requests to the OpenSearch hosts reusing the standard node-to-node communications, internally called the transport client.
```

Comment:
Its lovely to see the new world, but we dont know what it looks like for plugins. 
I would rather avoid talking about deprecation at this point because Plugins still add a lot of value and Java APIs might still be supported.

Reply:
Thanks removing the change to this paragraph.